tests/acceptance/features/bootstrap/WebDav.php

```diff
@@ -2390,8 +2390,9 @@ public function userUploadsAFileToWithAllMechanismsExceptNewChunking(
 			false,
 			'new'
 		);
-
-		$this->pushToLastStatusCodesArrays();
+		foreach ($this->uploadResponses as $response) {
+			$this->pushToLastHttpStatusCodesArray((string)$response->getStatusCode());
+		}
```

Comment:
the size of the response is fixed or its varies?

Reply:
it varies depending upon the combination of chunking method and dav version

Reply:
oh ok if so then we can check strictly.